src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,230 @@
+package duke.parser;
+
+import duke.command.*;
+import duke.task.ToDo;
+import org.junit.jupiter.api.Test;
+
+import static duke.parser.Parser.parse;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class ParserTest {
+    @Test
+    public void InvalidCommandTest(){
+        try {
+            assertFalse(parse("happy meal").isExit());
+            fail();
+        } catch (Exception exception) {
+            assertEquals("Please specify a valid command.",
+                    exception.getMessage());
+        }
+    }
+
```

Comment:
Is there a reason why you didn't add JavaDocs for the test cases? It can help explain what each test case is testing.